examples/jni/WORKSPACE

```diff
@@ -0,0 +1,57 @@
+workspace(name = "jni_example")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
Why should this be a separate workspace? If it were under the main workspace we would get build and test for free

Reply:
I followed the pattern I saw in `examples/testing/scalatest_repositories` and `examples/testing/specs2_junit_repositories/`. I thougth that the idea was to make the examples self-contained but I agree that it comes with the question of keeping them up to date.
Maybe this can be mitigated by using something like 
```
local_repository(
     name = "io_bazel_rules_scala",
     path = "../..",
)
```
?

Reply:
I think that those are tested by our e2es but I might be wrong. @liucijus are they tested?

Reply:
Also those are different since their WORKSPACE file is different (toolchain).
Is this different in any way?

Reply:
I don't think so. I'll put it in the main workspace